src/components/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Breadcrumbs = ({ parent, parentText, rendered }) => {
+  return (
+    <p className="breadcrumb">
+    <Link className="breadcrumb__item" to="/">
+      Home
+    </Link>
+    /
+    <Link className="breadcrumb__item" to={ `/${parent}` } >
+      { parentText }
+    </Link>
+    /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
+  </p>
+  )
```

Comment:
Spacing seems a bit off...

```suggestion
  return (
    <p className="breadcrumb">
      <Link className="breadcrumb__item" to="/">
        Home
      </Link>
      /
      <Link className="breadcrumb__item" to={ `/${parent}` } >
        { parentText }
      </Link>
      /<span className="breadcrumb__item breadcrumb__item--selected">{rendered}</span>
    </p>
  )
```